### PR TITLE
Update CHANGELOG.md for 0.16.0

### PR DESCRIPTION
**Pull request by pwang347**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+**Deprecation notice**
+* The mad aggregation type has been removed from the Group By and Aggregate operation as it is no longer available in Pandas.
+
+**Updates**
+* Improved support for detecting conditional formulas in the "by example" operations.
+* Improved keyboard accessibility by enabling keyboard focus on elements in the Cleaning Steps panel.
+* Improved keyboard accessibility in the Grid view by reducing the number of tab stops. Once focused, the grid can be navigated using the arrow keys, page up / down, home, etc.
+
+**Bugfixes**
+* Fixed issue where data frames with duplicate index keys could not be properly viewed or wrangled. Please note for this scenario that there are some limitations with generating the diff previews, these are outlined [here](https://github.com/microsoft/vscode-data-wrangler/wiki/Known-Issues).
```
Comment (kycutler) [inline CHANGELOG.md:13]:
```suggestion
* Fixed issue where data frames with duplicate index keys could not be properly viewed or wrangled. Please note for this scenario that there are some limitations with generating the diff previews, these are outlined [here](https://github.com/microsoft/vscode-data-wrangler/wiki/Known-Issues#pandas-dataframes-with-duplicate-index-keys).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## Version 0.16.0
+**Deprecation notice**
+* The mad aggregation type has been removed from the Group By and Aggregate operation as it is no longer available in Pandas.
```
Comment (kycutler) [inline CHANGELOG.md:5]:
```suggestion
* The `mad` (Mean Absolute Deviation) aggregation type has been removed from the Group By and Aggregate operation as it is no longer available in pandas.
```

**Review by kycutler — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## Version 0.16.0
+**Deprecation notice**
+* The `mad` (Mean Absolute Deviation) aggregation type has been removed from the Group By and Aggregate operation as it is no longer available in pandas.
+
+**Updates**
+* Improved support for detecting conditional formulas in the "by example" operations.
+* Improved keyboard accessibility by enabling keyboard focus on elements in the Cleaning Steps panel.
+* Improved keyboard accessibility in the Grid view by reducing the number of tab stops. Once focused, the grid can be navigated using the arrow keys, page up / down, home, etc.
```
Comment (jruales) [inline CHANGELOG.md:10]: nit: lowercase G in "Grid view"?

**Review by jruales — COMMENTED** [review]
LGTM

**Pull request merged**